icu4j/main/classes/core/src/com/ibm/icu/text/ListFormatter.java

```diff
@@ -348,4 +661,46 @@ private static ListFormatter load(ULocale ulocale, String style) {
     }
 
     static Cache cache = new Cache();
+
+    static String typeWidthToStyleString(ListFormatterType type, ListFormatterWidth width) {
+        switch (type) {
+            case AND:
+                switch (width) {
+                    case WIDE:
+                        return "standard";
+                    case SHORT:
+                        return "standard-short";
+                    case NARROW:
+                        return "standard-narrow";
+                    default:
```

Comment:
Can we remove `default:` ?
When we `Width` enum is added, we want Java compiler to warn it.

Reply:
Done.